docs/user-guide/building-assistants.rst

```diff
@@ -620,12 +675,38 @@ business logic. Read more about this `here <https://rasa.com/docs/rasa/core/form
 However, you should make sure not to handle any unhappy paths inside the form. These
 should be handled by writing regular stories, so your model can learn this behaviour.
 
+
+.. note::
+    Here's a minimal checklist of what you need to do to handle business logic using a form action:
+
+      - ``actions.py``: Define the form action
+              - Define the ``required_slots``, ``slot_mappings`` and ``submit`` methods
+      - ``data/nlu.md``: 
+          - Add examples for an intent to activate the form
+          - Add examples for an ``inform`` intent to fill the form
+      - ``domain.yml``: 
+          - Add all slots required by the form
+          - Add ``utter_ask_{slot}`` responses for all required slots
+          - Add your form action to the ``forms`` section 
+          - Add all intents and entities from your NLU training data
+      - ``data/stories.md``: 
+          - add a story for the form
+      - ``config.yml``: 
+          - Add the ``FormPolicy`` to your policies
+          - Add entity extractors & dimensions (according to which entities will fill your slots) to your pipeline
+      - ``endpoints.yml``: Define the ``action_endpoint``
+
+
+.. _handling-unexpected-user-input:
+
 Handling unexpected user input
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 All expected user inputs should be handled by the form we defined above, i.e. if the
 user provides the information the bot asks for. However, in real situations, the user
-will often behave differently. In this section we’ll go through various forms of
+will often behave differently.
+
+In this section we’ll go through various forms of
```

Comment:
why did we move that?

Reply:
I moved examples in and out of that section a few times and missed a backspace